x/bep3/abci.go

```diff
@@ -4,12 +4,14 @@ import (
 	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/kava-labs/kava/x/bep3/keeper"
+	"github.com/kava-labs/kava/x/bep3/types"
 )
 
 // BeginBlocker on every block expires outdated atomic swaps and removes closed
 // swap from long term storage (default storage time of 1 week)
-func BeginBlocker(ctx sdk.Context, k Keeper) {
-	if ctx.BlockTime().After(ModulePermissionsUpgradeTime) {
+func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
+	if ctx.BlockTime().After(types.ModulePermissionsUpgradeTime) {
 		err := k.EnsureModuleAccountPermissions(ctx)
 		if err != nil {
 			k.Logger(ctx).Error(fmt.Sprintf("couldn't update module account permissions: %v", err))
```

Comment:
This was added as a softfork to fix an issue after a chain upgrade last year; the bep3 mod account didn't have the correct permissions.
It could be removed, though it would be nice to validate account permissions somewhere.